dashboard/test/ui/features/acquisition_products/pd/teacher_application.feature

```diff
@@ -110,9 +110,7 @@ Scenario: Teacher starts a new application and submits it
 
   Then I press the first "input[name='committedToMasterSchedule']" element
   Then I press the first "input[name='replaceCourse']" element
-  Then I press the first "input[name='replaceWhichCourseCsd']" element
 
-  Then I press the first "input[name='committedToDiversity']" element
```

Comment:
Does this pass without adding the question that we can email you? It looks like that's required, so I hope it doesn't pass without that.

Reply:
Oh yes you're right! Thankks for catching this, this file changed a lot between the different branches and I think I mixed up the changes